src/Components/CreateImageWizard/utilities/certificates.ts

```diff
@@ -0,0 +1,5 @@
+export const isValidPEM = (cert: string): boolean => {
+  return /-----BEGIN CERTIFICATE-----[\s\S]+-----END CERTIFICATE-----/.test(
```

Comment:
Nit: isn't `[\s\S]+`  equal to `.+`?

Reply:
Hmmm, I tried to figure this out as well, we'd need to add "s" at the end of the regex, something like this:
`/-----BEGIN CERTIFICATE-----.+-----END CERTIFICATE-----/s`

Otherwise it would not accept certificates with more lines. I can do that if it makes the regex a bit more clear.